github-process-readme.md

```diff
@@ -0,0 +1,132 @@
+# FAQ – GITHUB Process 
+
+# 1. Configure ssh keys 
+
+A) Generate new ssh keys 
+```sh
+$ ssh-keygen -o -t rsa -b 4096 -C "<seagate-email-address>"  
+```
+B) Update github settings with this public ssh key
+  - Get id_rsa.pub key (default location: /root/.ssh/id_rsa.pub) 
+
+  - Go to github ssh key setting: https://github.com/settings/keys 
+
+  - Add new ssh key and then select Enable SSO option and Authorize for this key
+
+# 2. Git configuration process 
+
+A) Configure git 
+```sh
+$ git config --global user.name ‘Your Name’ 
+
+$ git config --global user.email ‘Your.Name@seagate.com’ 
+
+$ git config --global color.ui auto 
+```
+
+B) Clone s3server  
+
+a) using ssh: `$ git clone git@github.com:Seagate/cortx-s3server.git` 
```

Comment:
We may have to update this doc for cortx-test repo

Reply:
@seagate-sarang-sawant will do this in the next set of updates. Thanks!